frontend/src/app/dashboard/team.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, tap } from 'rxjs';
+
+export interface Team {
+  id: number;
+  name: string;
+}
+
+export interface OkrCycle {
+  cycle: string;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TeamService {
+  constructor(private httpClient: HttpClient) {}
+
+  public getTeams(): Observable<Team[]> {
+    return this.httpClient
+      .get<Team[]>('http://localhost:4200/api/v1/teams')
+      .pipe(tap((data) => console.log(data)));
+  }
+
+  public getQuarter(date = new Date()) {
```

Comment:
Das ist mit Abstand die grösste und komplizierteste  Codeblock der du in dieser Story geschrieben hat. Wieso testest du diese nicht ausführlich?
Zum Beispiel mit verschiedenen Tagen: erster Tag im Jahr, letzter Tag im Jahr ...
Dieser Test ist hat fast keine Abhängigkeiten zu Jest oder Karma. Der kannst du gut umsetzten.